lib/index.js

```diff
@@ -1,39 +1,55 @@
-"use strict";
-const fs = require("fs");
-const promisify = require("util.promisify");
-
+import fs from "fs";
+import promisify from "util.promisify";
+import imageminGifsicle from "imagemin-gifsicle";
+import imageminJpegtran from "imagemin-jpegtran";
+import imageminOptipng from "imagemin-optipng";
+import imageminSvgo from "imagemin-svgo";
+import imageminWebp from "imagemin-webp";
 const readFile = promisify(fs.readFile);
 const writeFile = promisify(fs.writeFile);
-
 const plugins = [
   [
-    "imagemin-gifsicle",
+    imageminGifsicle,
     {
-      interlaced: true
-    }
+      interlaced: true,
+    },
   ],
   [
-    "imagemin-jpegtran",
+    imageminJpegtran,
     {
-      progressive: true
-    }
+      progressive: true,
+    },
   ],
   [
-    "imagemin-optipng",
+    imageminOptipng,
     {
-      optimizationLevel: 5
-    }
+      optimizationLevel: 5,
+    },
   ],
   [
-    "imagemin-svgo",
+    imageminSvgo,
     {
-      plugins: [{ removeViewBox: false }]
-    }
-  ]
-].map(([name, opts]) => require(name)(opts));
-
-const minifyFile = (exports.minifyFile = filename =>
-  [...plugins, it => writeFile(filename, it)].reduce(
+      plugins: [
+        {
+          name: "removeViewBox",
+          active: false,
+        },
+      ],
+    },
+  ],
+  [
+    imageminWebp,
+    {
+      quality: 75,
```

Comment:
I think the idea behind imagemin-lint-staged was to have reasonable default settings that suit everyone. An arbitrary figure of 75% is a turn in the wrong direction. I would like to remove this.